roachpb/errors.go

```diff
@@ -342,18 +342,9 @@ func NewTransactionAbortedError() *TransactionAbortedError {
 }
 
 // NewTransactionPushError initializes a new TransactionPushError.
-// Txn is the transaction which will be retried. Both arguments are copied.
-// Transactions.
-func NewTransactionPushError(txn, pusheeTxn Transaction) *TransactionPushError {
-	err := &TransactionPushError{PusheeTxn: pusheeTxn.Clone()}
-	if txn.ID != nil {
```

Comment:
This behaviour has changed, and that's bad: Now a non-transactional pusher will get a half-baked transaction error back.